pkg/uuid/uuid.go

```diff
@@ -0,0 +1,42 @@
+/*
+Copyright 2014 The Kubernetes Authors.
```

Comment:
`2018 Authors of Cilium`? :)

Reply:
This is copied from the Kubernetes repo so the copyright is correct. I found this easier than creating a complicated go dependency for basic code like this.